include/chemfiles/File.hpp

```diff
@@ -90,6 +94,7 @@ class CHFL_EXPORT TextFile: public File, public std::iostream {
 
 private:
     void get_line(std::string& string);
+    void skip_line();
```

Comment:
`skip_line` and `skipline` seem very close in name to me. Is this intentional?

Reply:
yeah, this is somehow the same as `getline` and `get_line` =) I could rename them to `get_line_impl` and `skip_line_impl` if you prefer!

Reply:
Yeah, that's fine as long as the public API makes sense.